### PR TITLE
fix: hotfix tooltip position and admin buttons

### PR DESCRIPTION
**Pull request by rebelchris**
The position on the button was offset too much.
Used the old position of the report button.
Also hide if on desktop to only show on hover.
And added the moderation buttons inside.

### CODE REVIEW

File: packages/shared/src/components/PostOptionsMenu.tsx [hunk 1]
```diff
@@ -165,6 +171,20 @@ export default function PostOptionsMenu({
     text: 'Report',
     action: async () => setReportModal({ index: postIndex, post }),
   });
+  if (setShowDeletePost) {
+    postOptions.push({
+      icon: <MenuIcon Icon={TrashIcon} />,
+      text: 'Remove',
+      action: () => setShowDeletePost(),
```
Comment (idoshamun) [inline packages/shared/src/components/PostOptionsMenu.tsx:178]:
```suggestion
      action: setShowDeletePost,
```

File: packages/shared/src/components/PostOptionsMenu.tsx [hunk 2]
```diff
@@ -165,6 +171,20 @@ export default function PostOptionsMenu({
...
+    });
+  }
+  if (setShowBanPost) {
+    postOptions.push({
+      icon: <MenuIcon Icon={HammerIcon} />,
+      text: 'Ban',
+      action: () => setShowBanPost(),
```
Comment (idoshamun) [inline packages/shared/src/components/PostOptionsMenu.tsx:185]:
```suggestion
      action: setShowBanPost,
```

File: packages/webapp/pages/posts/[id].tsx [hunk 3]
```diff
@@ -844,7 +826,20 @@ const PostPage = ({ id, postData }: Props): ReactElement => {
           onRequestClose={() => setShowBanPost(false)}
         />
       )}
-      <PostOptionsMenu post={postData?.post} onMessage={onMessage} />
+      <PostOptionsMenu
+        post={postData?.post}
+        onMessage={onMessage}
+        setShowBanPost={
+          user?.roles?.indexOf(Roles.Moderator) > -1
```
Comment (idoshamun) [inline packages/webapp/pages/posts/[id].tsx:833]: Let's save it as a variable `isModerator` for better reuse

**Review by idoshamun — CHANGES_REQUESTED** [review]

**Comment by rebelchris** [comment]
@idoshamun Thanks! Cleaner indeed

**Review by idoshamun — APPROVED** [review]
Awesome!

**Pull request merged**